src/main/index.js

```diff
@@ -5,6 +5,7 @@ const { app, dialog, MessageChannelMain, ipcMain } = require('electron')
 const isDev = require('electron-is-dev')
 const contextMenu = require('electron-context-menu')
 const mkdirp = require('mkdirp')
+const createMapServer = require('@mapeo/map-server')
```

Comment:
since we're doing TS checking in this file, would update this import to:

```suggestion
const createMapServer = require('@mapeo/map-server').default
```

That way you get the proper TS annotation (see https://github.com/digidem/mapeo-map-server#usage)

Reply:
Im not sure why, but adding `.default` just breaks the app. 
<img width="688" alt="image" src="https://user-images.githubusercontent.com/67773827/207977925-86a8f6ce-3e9f-4261-8f81-fa32d2a1f31a.png">

Reply:
ah yeah that's on the map server side then, we may have removed that functionality for some reason and didn't update the docs. nevermind then! 😄